django/db/migrations/loader.py

```diff
@@ -285,6 +285,24 @@ def build_graph(self):
             raise
         self.graph.ensure_not_cyclic()
 
+    def check_consistent_migration_files(self, connection, app_labels):
```

Comment:
Do we need `app_labels`? IMO it should be fine to check all apps like in `check_consistent_history()`.

Reply:
I will commit the desired changes.